### PR TITLE
build: docker images

### PR DESCRIPTION
**Pull request by Brooooooklyn**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ pnpm-lock.yaml
 target
 lib
 test-results
+packages/graphql/src/graphql/index.ts
```
Comment (himself65) [inline .prettierignore:5]: what is this

**Review by himself65 — CHANGES_REQUESTED** [review]

File: .prettierignore (hunk 1 above)
Comment (Brooooooklyn) [reply .prettierignore:5]: graphql-codegen generated file

Comment (himself65) [reply .prettierignore:5]: but you actually emit that file in source code

Comment (himself65) [reply .prettierignore:5]: https://github.com/toeverything/AFFiNE/pull/2860/files#diff-cba3e7481bb9a82af6806d6bf71bfc5624df5000ac660e73a3bde48904d70046

**Review by darkskygit — APPROVED** [review]

**Pull request merged**